packages/ruleset-bundler/src/loader/common/bundle.ts

```diff
@@ -0,0 +1,33 @@
+import { migrateRuleset } from '@stoplight/spectral-ruleset-migrator';
+import * as path from '@stoplight/path';
+import { ErrorWithCause } from 'pony-cause';
+
+import { stdin } from '../../plugins/stdin';
+import { isBasicRuleset, isErrorWithCode } from './utils';
+import { bundleRuleset, BundleOptions, IO } from '../../index';
+
+export async function bundle(rulesetFile: string, bundleOptions: BundleOptions, { fs }: IO): Promise<string> {
+  try {
+    if (isBasicRuleset(rulesetFile)) {
+      const migratedRuleset = await migrateRuleset(rulesetFile, {
```

Comment:
Ah I think I see - `migrateRuleset` naming confused me. More like a ruleset transformer in this case, to transform from plain object format to esm format.